### PR TITLE
texstudio: New package

### PR DESCRIPTION
**Pull request by podsvirov**
We have [texlive](https://packages.msys2.org/search?q=texlive) and what about to appropriate editor like [TeXstudio](https://www.texstudio.org/)?

I apply several patches and will try to sugest upstream them.

Cc: @naveen521kk @jeremyd2019 (please review)

### CODE REVIEW

**Comment by Biswa96** [comment]
Can quazip-qt6 addition help any features?

**Comment by podsvirov** [comment]
> Can quazip-qt6 addition help any features?

This will be a nice improvement to the distribution. The built-in [quazip](https://github.com/msys2/MINGW-packages/runs/7101639923?check_suite_focus=true#step:9:425) is now used.

File: mingw-w64-texstudio/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+_realname=texstudio
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=4.2.3
+pkgrel=1
+pkgdesc="Integrated writing environment for creating LaTeX documents (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
+url="http://texstudio.sourceforge.net/"
+license=('GPL')
```
Comment (naveen521kk) [inline mingw-w64-texstudio/PKGBUILD:12]:
```suggestion
license=('spdx: GPL-2.0-or-later')
```

File: mingw-w64-texstudio/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Maintainer: Konstantin Podsvirov <konstantin@podsvirov.pro>
+
+_realname=texstudio
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=4.2.3
+pkgrel=1
+pkgdesc="Integrated writing environment for creating LaTeX documents (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
```
Comment (naveen521kk) [inline mingw-w64-texstudio/PKGBUILD:10]: Does this compile on arm?

**Comment by Biswa96** [comment]
I have compiled texstudio with latest quazip+qt6 package and it builds and works without any issue. quazip update will be in the pacman repository.

File: mingw-w64-texstudio/PKGBUILD (hunk 2 above)
Comment (podsvirov) [reply mingw-w64-texstudio/PKGBUILD]: Not sure - removed.

**Comment by podsvirov** [comment]
> I have compiled texstudio with latest quazip+qt6 package and it builds and works without any issue. quazip update will be in the pacman repository.

Thanks!

Just for notes: the `tulip` package have `quazip` (that will be removed) in depends.

**Comment by Biswa96** [comment]
qazip is not built yet. See this pending page https://packages.msys2.org/queue. Once it is completed, the CI can be restarted here.

**Pull request merged**